src/app/app.component.ts

```diff
@@ -45,11 +52,25 @@ export class AppComponent extends LogoutBasePageComponent implements OnInit {
     await this.initialisePersistentStorage();
     this.store$.dispatch(LoadAppVersion());
     await this.configureStatusBar();
+    if (this.platform.is('cordova')) {
+      // @TODO MES-6268 enable this code
+      // this.configureAccessibility();
+      this.configurePlatformSubscriptions();
+    }
     await this.disableMenuSwipe();
     await SplashScreen.hide();
     this.logoutEnabled$ = this.store$.select(selectLogoutEnabled);
   }
 
+  ionViewWillUnload() {
+    if (this.platformSubscription) {
+      this.platformSubscription.unsubscribe();
+    }
+    if (this.subscription) {
+      this.subscription.unsubscribe();
+    }
```

Comment:
is this subscription needed?

Reply:
yeah agree it appears redundant not sure why it was in original code. Will remove